charts/garnet/Chart.yaml

```
@@ -1,20 +1,27 @@
 apiVersion: v2
 name: garnet
-description: A Helm chart for Microsoft garnet
+description: A Helm chart for Microsoft Garnet
 type: application
-version: 0.1.2
-appVersion: Version.props
+version: 0.1.3
+appVersion: "Version.props"
```

remove the double quotes here, as we already added it via `sed` above.
```suggestion
appVersion: Version.props
```

---

@badrishc Not exactly. The double quotes in the sed command is to ensure that the substitution will take the variable.

---

if you commit my suggestion for the change to `sed` then these quotes can be eliminated.

---

I tried this and it replaced Version.props with "1.2.3" including adding the quotes:

`sed -i -e 's#Version.props#"'${VERSION_PROPS}'"#g' charts/garnet/Chart.yaml`

Because the double quote is within the single quote, so it gets rendered by `sed`

---

My bad ...